pyuvdata/tests/test_miriad.py

```diff
@@ -674,3 +674,30 @@ def test_antpos_units():
     aantpos = (uvutils.ECEF_from_rotECEF(aantpos, uv.telescope_location_lat_lon_alt[1]) -
                uv.telescope_location)
     nt.assert_true(np.allclose(aantpos, uv.antenna_positions))
+
+
+def test_readMiriadwriteMiraid_check_time_format():
+    """
+    test time_array is converted properly from Miriad format
+    """
+    # test read-in
+    fname = os.path.join(DATA_PATH, 'zen.2457698.40355.xx.HH.uvcA')
+    uvd = UVData()
+    uvd.read_miriad(fname)
+    uvd_t = uvd.time_array.min()
+    uvd_l = uvd.lst_array.min()
+    uv = aipy.miriad.UV(fname)
+    uv_t = uv['time'] + uv['inttime'] / (24 * 3600.) / 2
+    uv_l = uv['lst'] + uv['inttime'] * 2 * np.pi / (23.9344699 * 3600.) / 2
```

Comment:
I've never really like hardcoding values like hours in a sidereal day.  Obviously looks like you've got a lot of decimal places, but any idea how this precision compares with some of our other tolerances?